app/services/iex_service.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class IEXService
+  BASE_URL = 'https://api.iextrading.com/1.0'.freeze
+  COMPANY_SYMBOLS_ENDPOINT = 'ref-data/symbols'.freeze
+
+  def self.search_stocks(company_name)
+    search_url = "#{BASE_URL}/#{stocks_endpoint(company_name)}"
+
+    stock = HTTParty.get search_url
+    stock.parsed_response
+  end
+
+  def self.stocks_endpoint(company_name)
```

Comment:
This is going to be a problem -- need to make it much more resilient before showtime. Probably a fuzzy match on the find and some kind of 'no company found, suggestions?' option will work.